src/mixins/filing-mixin.ts

```diff
@@ -1,14 +1,16 @@
 import { Component, Vue } from 'vue-property-decorator'
-import { mapActions, mapState } from 'vuex'
-import { FilingDataIF } from '@/interfaces'
-import { LegalTypes, FilingCodes } from '@/enums'
+import { mapActions, mapGetters, mapState } from 'vuex'
+import { CorrectionFilingIF, FilingDataIF } from '@/interfaces'
+import { FilingTypes, LegalTypes, FilingCodes } from '@/enums'
+import { IncorporationFilingIF } from '@/interfaces/incorporation-interfaces'
 
 /**
  * Mixin that provides some useful filing utilities.
  */
 @Component({
   computed: {
-    ...mapState(['filingData', 'entityType'])
+    ...mapState(['filingData', 'entityType']),
+    ...mapGetters(['getCurrentDate'])
```

Comment:
There's already a state to get currentDate.
(But if you wanted to change it to a getter, I'm in favour.)

Reply:
Yea, you sold me on getters awhile ago! Lol Lets stick with em